chapter15/qli/ex4.pl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+use 5.010001;
+
+given ( $ARGV[0] ) {
+  when ( undef ) { print "Please input a number as cmd argument! \n"; }
```

Comment:
this is check for `null` argument, the next check is for Number validation. I think Number Validation will have warnings for null values, if I don't gate with the null value check.